From 774f6f1c151ae9d84d0d3da0e32d2cea6c09799f Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Wed, 6 Jun 2012 13:14:46 +0800
Subject: [PATCH 1574/2463] ENGR00212512: Fix caam build error when being
 built as module

Fix caam build error when being built as module.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 drivers/crypto/caam/caamalg.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/crypto/caam/caamalg.c b/drivers/crypto/caam/caamalg.c
index 3c3e76b..549d526 100644
--- a/drivers/crypto/caam/caamalg.c
+++ b/drivers/crypto/caam/caamalg.c
@@ -2180,6 +2180,7 @@ void caam_algapi_shutdown(struct platform_device *pdev)
 	}
 	kfree(priv->algapi_jr);
 }
+EXPORT_SYMBOL_GPL(caam_algapi_shutdown);
 
 static struct caam_crypto_alg *caam_alg_alloc(struct device *ctrldev,
 					      struct caam_alg_template
@@ -2283,6 +2284,7 @@ int caam_algapi_startup(struct platform_device *pdev)
 
 	return err;
 }
+EXPORT_SYMBOL_GPL(caam_algapi_startup);
 
 #ifdef CONFIG_OF
 static void __exit caam_algapi_exit(void)
-- 
1.7.7.4

